.bazelrc

```diff
@@ -94,6 +94,8 @@ build:release-debug-common --config=forcedebug
 build:release-debug-common --config=debugsymbols
 build:release-debug-common --config=skipslowenforce
 
+build:untyped-blame --copt=-DTRACK_UNTYPED_BLAME_MODE
```

Comment:
Should this also be `blame-untyped`, so that the word order isn't flipped from the command line option?

Reply:
Want to leave a comment here about why we have this (e.g., how it would be too slow to do all the time, but doing it in debug mode is also too slow on large codebases)?

Or if you want to leave that comment somewhere else in the PR, feel free.